<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAuthDlg</class>
 <widget class="QDialog" name="CAuthDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>149</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>授权窗口</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>加密算法</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbNoEnc">
        <property name="text">
         <string>不加密</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbSM4">
        <property name="text">
         <string>SM4</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>密钥：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePassword">
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbConfirm">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbConfirm</sender>
   <signal>clicked()</signal>
   <receiver>CAuthDlg</receiver>
   <slot>sltConfirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSM4</sender>
   <signal>toggled(bool)</signal>
   <receiver>CAuthDlg</receiver>
   <slot>sltSM4(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbNoEnc</sender>
   <signal>toggled(bool)</signal>
   <receiver>CAuthDlg</receiver>
   <slot>sltNoEnc(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sltConfirm()</slot>
  <slot>sltNoEnc(bool)</slot>
  <slot>sltSM4(bool)</slot>
 </slots>
</ui>
